luigi/s3.py

```diff
@@ -281,7 +281,7 @@ def get_as_string(self, s3_path):
 
         return contents
 
-    def copy(self, source_path, destination_path, **kwargs):
+    def copy(self, source_path, destination_path, part_size=67108864, **kwargs):
```

Comment:
Yeah, this can probably be removed from here and moved into a var within the function.